src/js/components/Overview/ChartCard.tsx

```diff
@@ -30,6 +29,7 @@ const ChartCard: React.FC<ChartCardProps> = memo(({ section, chart, onRemoveChar
   const td = useTranslationDefault();
   const containerRef = useRef<HTMLDivElement>(null);
   const width = useElementWidth(containerRef, chart.width);
+  const CARD_STYLE: React.CSSProperties = { height: '415px', borderRadius: '11px', ...BOX_SHADOW };
```

Comment:
why was this moved?

Reply:
remnant of a removed change. now back in position.